.devcontainer/cpp/Dockerfile

```diff
@@ -59,7 +60,19 @@ RUN apt-get update && apt-get install -y --no-install-recommends libclang-${CLAN
  && CC=clang CXX=clang++ cmake -S /tmp/mull -B /tmp/mull/build \
  && cmake --build /tmp/mull/build --target install \
  && rm -rf /tmp/mull \
- && apt-get purge -y libclang-${CLANG_VERSION}-dev llvm-${CLANG_VERSION}-dev libsqlite3-dev \
+ && wget -qO - https://github.com/bkryza/clang-uml/archive/refs/tags/${CLANG_UML_VERSION}.tar.gz | tar xz -C /tmp \
+ && CC=clang CXX=clang++ cmake -S /tmp/clang-uml-${CLANG_UML_VERSION} -B /tmp/clang-uml-${CLANG_UML_VERSION}/release \
+ -DCMAKE_BUILD_TYPE=Release \
+ -DCUSTOM_COMPILE_OPTIONS=-Wno-sign-compare \
+ -DLLVM_VERSION=${CLANG_VERSION} \
+ -DLINK_LLVM_SHARED=ON \
+ -DENABLE_CUDA_TEST_CASES=OFF \
+ -DENABLE_CXX_MODULES_TEST_CASES=OFF \
+ --compile-no-warning-as-error\
+ && cmake --build /tmp/clang-uml-${CLANG_UML_VERSION}/release --config release --target clang-uml \
+ && cmake --install /tmp/clang-uml-${CLANG_UML_VERSION}/release \
```

Comment:
In principle these two steps can be merged. The `install` target should already build all its dependencies before installing.

Reply:
If I remember correctly, I had to choose target to avoid building unnecessary stuff. It seems this cannot be passed with install. So I'll keep it.